### DISCUSSION TITLE
how to get number of times an item in collection is viewed ?

### DISCUSSION THREAD

**Discussion by mohanbonam** [directus/directus#15835] [Archive]
created a feild num_of_views in a collection
how to update num_of_views in each item in a collection, when fetched by item id "baseURL/some_collection/:id"?

**Comment by br41nslug** [comment]
This is not something that is tracked normally but it should be possible to track manually using either a Flow or custom hook. Just beware that this number might get up there pretty fast when just browsing the app.
